OrbitQt/TutorialOverlay.cpp

```diff
@@ -86,12 +85,43 @@ QRect TutorialOverlay::AbsoluteGeometry(QWidget* widget) const {
   QRect result = widget->geometry();
   QRect parent_geo = AbsoluteGeometry(parent_widget);
 
-  result.moveTo(parent_geo.topLeft());
+  result.setTopLeft(result.topLeft() + parent_geo.topLeft());
+  result.setBottomRight(result.bottomRight() + parent_geo.topLeft());
 
   return result;
 }
 
-TutorialOverlay::~TutorialOverlay() {}
+void TutorialOverlay::DeleteProgressIndicators() {
+  auto layout = ui_->progressLayout;
+  int count = layout->count() - 1;
+  for (int i = 1; i < count; ++i) {
```

Comment:
Is there an off by one here? i starts at 1 and count is "layout->count() - 1" . Is that wanted?

Reply:
This is correct, the layout contains two horizontal spacers and the radio buttons are added in between. I should probably add a comment to explain that as it looks very suspicious.